bin/rhts-sync-block

```diff
@@ -135,19 +148,19 @@ def main():
             timeout = datetime.datetime.now() + datetime.timedelta(seconds=int(val))
 
     machines = args[0:]
-    machines = filter(isset,machines)
+    machines = list(filter(isset,machines))
```

Comment:
You sure this list should stay since you've removed it elsewhere.

Reply:
This one is correct, in py3 `filter` works as generator so we have to explicitly create list from it